Havit.Blazor.Components.Web.Bootstrap/wwwroot/HxMultiSelect.js

```diff
@@ -0,0 +1,50 @@
+﻿export function initialize(element, hxMultiSelectDotnetObjectReference, shouldToggle) {
+	if (!element) {
+		return;
+	}
+
+	element.hxMultiSelectDotnetObjectReference = hxMultiSelectDotnetObjectReference;
+	element.addEventListener('shown.bs.dropdown', handleDropdownShown);
+	element.addEventListener('hidden.bs.dropdown', handleDropdownHidden);
+
+	var d = new bootstrap.Dropdown(element, {
+		toggle: shouldToggle,
+	});
+}
```

Comment:
Please remove the `shouldToggle` option, it is always `false`.

Reply:
Have removed the toggle option